packages/data-context/src/sources/CloudDataSource.ts

```diff
@@ -264,10 +299,27 @@ export class CloudDataSource {
     `, { args }, {
       fetchOptions: {
         headers: {
-          // TODO: replace this with an exhange to filter out this request
+          // TODO: replace this with an exchange to filter out this request
           INTERNAL_REQUEST: JSON.stringify({ data: { _cloudCacheInvalidate: true } }),
         },
       },
     }).toPromise()
   }
+
+  async getCache () {
+    await this.#cloudUrqlClient.mutation(`
+      mutation Internal_showUrqlCache { 
+        _showUrqlCache
+      }
+    `, { }, {
+      fetchOptions: {
+        headers: {
+          // TODO: replace this with an exchange to filter out this request
```

Comment:
Are the TODO's in this file pending for this effort, or are these future cleanup items?

Reply:
I think this is meant for future work. Right, @tgriesser?

Reply:
It would be ideal for the TODOs to be linked to a GitHub issue so they don't fall into the ever-expanding black hole of forgotten TODOs across the universe 😆 ⚫